src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -843,4 +873,61 @@ private function mergeParameter(Parameter $actual, Parameter $defined): Paramete
 
         return $actual;
     }
+
+    /**
+     * @param string[]              $errors
+     * @param array<string, string> $responseMimeTypes
+     */
+    private function addOperationErrors(Operation $operation, array $errors, array $responseMimeTypes, ResourceMetadataCollection $resourceMetadataCollection, Schema $schema, \ArrayObject $schemas): Operation
+    {
+        $existingResponses = null;
+        foreach ($errors as $error) {
+            if (!is_a($error, ProblemExceptionInterface::class, true)) {
+                $this->logger?->warning(\sprintf('The error class "%s" does not implement "%s". Did you forget a use statement?', $error, ProblemExceptionInterface::class));
+            }
+
+            $status = null;
+            $description = null;
+            try {
+                /** @var ProblemExceptionInterface */
+                $exception = (new \ReflectionClass($error))->newInstanceWithoutConstructor();
+                $status = $exception->getStatus();
+                $description = $exception->getTitle();
+            } catch (\ReflectionException) {
+            }
+
+            try {
+                $errorOperation = $this->resourceMetadataFactory->create($error)->getOperation();
+                if (!is_a($errorOperation, Error::class)) {
+                    $this->logger?->warning(\sprintf('The error class %s is not an ErrorResource', $error));
+                    continue;
+                }
+                /* @var Error $errorOperation */
+                $status ??= $errorOperation->getStatus();
+                $description ??= $errorOperation->getDescription();
+            } catch (ResourceClassNotFoundException|OperationNotFoundException) {
+                $this->logger?->warning(\sprintf('The error class %s is not an ErrorResource', $error));
```

Comment:
Same here. It's probably useful to not catch this exception and be sure the developer handles this case.